yacc/console/gram.y

```diff
@@ -421,18 +427,18 @@ dataspace_define_stmt:
 	}
 
 sharding_rule_define_stmt:
-	SHARDING RULE any_id sharding_rule_table_clause sharding_rule_argument_list
+	SHARDING RULE any_id link_dataspace sharding_rule_table_clause sharding_rule_argument_list
```

Comment:
idk maybe
`CREATE SHARDING RULE r1 DATASPACE ds1` is not sqlish enough. 
IMO `CREATE SHARDING RULE r1 USING DATASPACE ds1` or `CREATE SHARDING RULE r1 FOR DATASPACE ds1` would be better

Reply:
Changed. Now declaring:
CREATE SHARDING RULE r1 FOR DATASPACE ds1